packages/libraries/client/README.md

```diff
@@ -304,3 +304,30 @@ const server = new ApolloServer({
   ]
 })
 ```
+
+## Self-Hosting
+
+To align the client with your own instance of GraphQL Hive, you should use `selfHosting` options in the client configuration.
```

Comment:
Would it be valuable to explain that the `selfHosting` options take precedence over the legacy ones `options.hosting.endpoint` and `options.usage.endpoint`
I believe we might want to remove the legacy ones at some point later via breaking change? Unless there is some other reason for keeping them I'm not aware of, but in either case might be beneficial to explain it so newcomers are not confused